Wrappers/Python/ccpi/viewer/CILViewer.py

```diff
@@ -604,21 +610,20 @@ def installPipeline(self):
 
         self.adjustCamera()
 
-        self.saveDefaultCamera()
-
         self.iren.Initialize()
         self.renWin.Render()
 
     def saveDefaultCamera(self):
-        camera = vtk.vtkCamera()
-        camera.SetFocalPoint(self.getCamera().GetFocalPoint())
-        camera.SetPosition(self.getCamera().GetPosition())
-        camera.SetViewUp(self.getCamera().GetViewUp())
-        self.default_camera = camera
+        ''' Saves the default camera settings for a particular
+        loaded 3D image.'''
+        self.default_camera_data = CameraData(self.getCamera())
 
     def resetCameraToDefault(self):
-        if hasattr(self, 'default_camera'):
-            self.ren.SetActiveCamera(self.default_camera)
+        ''' resets to the default camera settings for the current
+        loaded 3D image'''
+        if hasattr(self, 'default_camera_data'):
+            self.adjustCamera(resetcamera=True)
+            CameraData.CopyDataToCamera(self.default_camera_data, self.getCamera())
```

Comment:
I'm not particularly fond of this method name, but I could live with it.

Reply:
I have added docstrings